### ISSUE TITLE
[Suggestion] Quick Value Adjustment in UI

### ISSUE THREAD

**Issue by wagonbomb** [thommcgrath/Beacon#60]
If there were a way to adjust quantities and qualities of items in the list view, the speed of creating a completely custom drop table would be MASSIVELY increased. I thought of this as I have been going through creating something for my PvP cluster since I can't seem to find anything even remotely balanced. 

Maybe a dropdown for qualities and entry box for quantities would be a great solution. You could even include the item weights as a numerical value, thus creating the need to enter the detailed properties window only when you want to add multiple items. 

![example](https://user-images.githubusercontent.com/31332239/29698658-3bcb4cba-8925-11e7-9be0-78fdacc1915e.PNG)

I'm not knowledgeable in coding so I don't know what coding would be involved in this solution, however I worked in QC and beta testing at a company for some years and understand logic/workflow. If there's immediate way I can see to optimize the UI I think it would be this. The fact that you've even gone and created this much of an interface is absolutely great. I wouldn't have taken on the task of creating my own drop table were it not for this. Just thought I'd give you my own input since I really like what you've got here. :)

**Comment by thommcgrath** [comment]
Unfortunately, the issue at hand is my choice of tool. Xojo has a lot of strengths, but its ListBox control is not one of them. While it does support editable cells, even after 20 years of development, it really doesn't feel natural. Clicking has to happen in exactly the right spot, such as the leftmost 10 pixels of the cell, and requires something like a slow double-click. It's always awkward.

I've used this in the engram selection list to control weights for lack of a better option, but I really hate doing so. I just don't think it feels good enough to use in the code UI.

**Comment by wagonbomb** [comment]
Does it support any type of CSV import? I've done database migrations before via CSV I'm sure I could handle some drops lol.

**Comment by thommcgrath** [comment]
No, there isn't any kind of CSV import or export. I don't have an immediate option for you, but I'll see what I can come up with.
